internal/venice-common/src/main/proto/VeniceReadService.proto

```diff
@@ -29,4 +30,21 @@ message VeniceServerResponse {
 
   uint32 errorCode = 6;
   string errorMessage = 7;
+}
+
+message CountByValueRequest {
+  repeated bytes keys = 1;
+  string resourceName = 2;
+  int32 topK = 3;
+  repeated string fieldNames = 4;
```

Comment:
Just curious do we need a field of isRetryRequest for this one, if it is just between client and server?